src/main/webapp/app/shared/date-time-picker/date-time-picker.component.ts

```diff
@@ -44,6 +51,7 @@ export class FormDateTimePickerComponent implements ControlValueAccessor {
     @Input() startAt: dayjs.Dayjs = dayjs().startOf('minutes'); // Default selected date. By default this sets it to the current time without seconds or milliseconds;
     @Input() min: dayjs.Dayjs; // Dates before this date are not selectable.
     @Input() max: dayjs.Dayjs; // Dates after this date are not selectable.
+    @Input() timeZoneWarning: boolean; // Displays a warning that the current time zone might differ from the participants'.
```

Comment:
[nit] maybe rename it to `shouldShowTimeZoneWarning`?

Reply:
Thank you for the review and the feedback, I implemented it.